src/multicast/multicast.go

```diff
@@ -80,8 +86,14 @@ func (m *Multicast) Start() error {
 	return nil
 }
 
+// IsStarted returns true if the module has been started.
+func (m *Multicast) IsStarted() bool {
+	return m.isOpen
```

Comment:
I just noticed this when `isStarted` was added, but I think even the existing use of `isOpen` (and possible some other multicast functions) may not be thread safe. This function certainly is not, since `IsStarted` is both usable outside the library and called within the actor in some of the code below. We probably need the exported `IsStarted` code to use a `phony.Block` to check this flag, and then skip the function and check the flag directly when we need it from inside the actor.